### PR TITLE
MNIST Fashion dataset example

### PR DESCRIPTION
**Pull request by gogasca**
Implements a simple DNN for MNIST Fashion dataset.
This is the first example of a series of models which will be added.
This is an initial example of a series of models which will improve the accuracy.
Original code: https://www.tensorflow.org/tutorials/keras/basic_classification

### CODE REVIEW

**Comment by gogasca** [comment]
Created https://github.com/GoogleCloudPlatform/cloudml-samples/issues/274
